docs/pages/ESSupport.md

```diff
@@ -854,3 +854,45 @@ This analyzer checks every JavaScript file for occurences of a `sap/ui/core/Core
 ### JSDoc
 
 The UI5 Tooling offers a JSDoc build, which is enhanced by UI5 specific JSDoc features.
+
+Following artefacts are part of the JSDoc build:
+
+- [jsdocGenerator.js](https://github.com/SAP/ui5-builder/blob/main/lib/processors/jsdoc/jsdocGenerator.js)
+  Executes the actual JSDoc build and creates the `api.json`.
+- [sdkTransformer.js](https://github.com/SAP/ui5-builder/blob/main/lib/processors/jsdoc/sdkTransformer.js)
+  Transforms the `api.json` for usage in a UI5 SDK.
+- [apiIndexGenerator.js](https://github.com/SAP/ui5-builder/blob/main/lib/processors/jsdoc/apiIndexGenerator.js)
+  Compiles API index resources from all `api.json` resources available in the given test resources directory.
+  The resulting index resources (e.g. `api-index.json`,  `api-index-deprecated.json`,
+  `api-index-experimental.json` and `api-index-since.json`) are mainly to be used in the UI5 SDK.
+
+#### JSDoc: UI5 SDK
```

Comment:
I wonder about the differences of the sections `JSDoc` and `JSDoc: UI5 SDK`. The first lists all relevant processors and this one lists all scripts used by them. To me those two sections are a bit confusing.

Reply:
true. I had placed all files located in the `ui5` subfolder in this section. do you or @codeworrior know what was the reason for introducing this subfolder?

Reply:
The subfolder contains the files that are unrelated to UI5 Tooling and originally part of the OpenUI5 repository.